examples/guides/mp-restful-webservice/README.adoc

```diff
@@ -669,223 +499,210 @@ accepting it. But omitting that validation lets us create an easy health check
 that we can use by simply setting the greeting to blank from
 a `curl` command.
 
-=== Add a new JAX-RS resource class
-Create `HealthResource.java` to define the endpoints for checking if the service is
+Helidon MicroProfile provides a built-in health framework. We can tap into that
+easily to add our app-specific liveness check.
+
+=== Add a new class for the health check
+Create `CheckLiveness.java` to define the endpoints for checking if the service is
 active and if it is ready.
```

Comment:
nit: ...checking whether the service is active and whether it is ready.

Reply:
done